core/CMakeLists.txt

```diff
@@ -2,13 +2,11 @@ message(STATUS "Building 'actsvg::core' component")
 
 # Set up the library.
 actsvg_add_library( actsvg_core core
-   "include/actsvg/actsvg.hpp"
-   "include/actsvg/core/connectors.hpp"
-   "include/actsvg/core/defs.hpp"
-   "include/actsvg/core/draw.hpp"
-   "include/actsvg/core/generators.hpp"
-   "include/actsvg/core/style.hpp"
-   "include/actsvg/core/svg.hpp"
-   "include/actsvg/core/utils.hpp"
-   "include/actsvg/core/views.hpp")
-
+   "src/core/connectors.cpp"
+   "src/core/defs.cpp"
+   "src/core/draw.cpp"
+   "src/core/generators.cpp"
+   "src/core/style.cpp"
+   "src/core/svg.cpp"
+   "src/core/utils.cpp"
```

Comment:
It would still be a good idea to keep listing the headers. IDEs can take advantage of that.

Reply:
Restored.